### DISCUSSION TITLE
Please may I join you team

### DISCUSSION THREAD

**Discussion by NRanjan-17** [ROM-builders/temporary#9565] [General]
Please  sir may I join your team?
Now I know all things like-Cherry-pick, rebase, git commit and other things.
Please Ignore my previous worst commits.
https://github.com/NRanjan-17/device_xiaomi_mido_pixel
https://github.com/NRanjan-17/proprietary_vendor_xiaomi
https://cirrus-ci.com/github/ROM-builders/temporary/pull/9470
https://cirrus-ci.com/github/ROM-builders/temporary/pull/9435
https://cirrus-ci.com/github/ROM-builders/temporary/pull/8603

**Comment by Apon77** [comment]
Please read #learn instruction from telegram group

**Reply by NRanjan-17** [reply]
> Please read #learn instruction from telegram group

Ok sir
But on what should i focus more

**Reply by NRanjan-17** [reply]
> Please read #learn instruction from telegram group

I also want to show a concern to you , sir you are doing a great job of Rom-Builders. But sir I want to ask one think from you 
In group in approval section it is written as-
"If schedule time is more than 10h, it makes it auto PR block, if less than 2h, it auto removes the block. You need to wait until the block removes. And after removing the block you need to do PR again for starting a build."
But my build was scheduled under 10h and build time was less than 1 hour but then also I am getting Needs approval
and this Needs Approval is only removed when whole queue is complete but who will it be completed when your teammates are starting there builds constantly one after another and without proper PR label .
Please do something on it

**Reply by NRanjan-17** [reply]
https://cirrus-ci.com/build/6344712700297216

**Reply by Apon77** [reply]
> > Please read #learn instruction from telegram group
> 
> I also want to show a concern to you , sir you are doing a great job of Rom-Builders. But sir I want to ask one think from you In group in approval section it is written as- "If schedule time is more than 10h, it makes it auto PR block, if less than 2h, it auto removes the block. You need to wait until the block removes. And after removing the block you need to do PR again for starting a build." But my build was scheduled under 10h and build time was less than 1 hour but then also I am getting Needs approval and this Needs Approval is only removed when whole queue is complete but who will it be completed when your teammates are starting there builds constantly one after another and without proper PR label . Please do something on it

they are not required to use pr label

**Reply by NRanjan-17** [reply]
Ok sir
What things I need to improve more to become a teammate

**Reply by Apon77** [reply]
> Ok sir What things I need to improve more to become a teammate
https://t.me/ROM_builders/80728
